test/controllers/WebTest.java

```diff
@@ -163,4 +167,15 @@ private void checkSparkJobExceptionDetailElementForFlow2(JsonElement jobExceptio
 
     }
   }
-}
\ No newline at end of file
+
+  private void checkTonyJobExceptionDetailElementForFlow2(JsonElement jobExceptionDetail) {
+    JsonObject jobExceptionJsonObject = jobExceptionDetail.getAsJsonObject();
+    Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.NAME).getAsString(), "job_name_2_3");
+    Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.ID).getAsString(), "job_name_2_3");
+    Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.TYPE).getAsString(), "TONY");
+    JsonArray application = jobExceptionJsonObject.getAsJsonArray(JsonKeys.APPLICATIONS);
+    JsonObject applicationJsonObject = application.get(0).getAsJsonObject();
+    Assert.assertEquals("Not a TONY_INFRA_ERROR ",
+        applicationJsonObject.get(JsonKeys.EXCEPTION_CLASSIFICATION).getAsString(), "TONY_INFRA_ERROR");
```

Comment:
(Same comment for line 131)
The usage of assertEquals is incorrect.
It should be used as `assertEquals(String message, Object expected, Object actual)` instead of assertEquals(String message, Object actual, Object expected)

Reply:
I don't understand this , its  similar to any other assert statement .  for e.g Assert.assertEquals(jobExceptionJsonObject.get(JsonKeys.NAME).getAsString(), "job_name_2_1");. 
Can you explain with example